Notebooks/DC2_Postage_Stamps.ipynb

```diff
@@ -104,7 +141,7 @@
     "    xy = afwGeom.PointI(tractInfo.getWcs().skyToPixel(radec))\n",
     "    bbox = afwGeom.BoxI(xy - cutoutSize//2, cutoutSize)\n",
     "\n",
-    "    coaddId = {'tract': tractInfo.getId(), 'patch': \"%d,%d\" % patchInfo.getIndex(), 'filter': filt}\n",
+    "    coaddId = {'tract': tractInfo.getId(), 'patch': \"%d,%d\" % patchInfo.getIndex(), 'filter': filter}\n",
```

Comment:
I looked at the "filter"s and "filt"s for a while and then decided it would be easier if we always just used "filter", always.

Reply:
I've been trying to train myself to type `filt` (because `filter` is a reserved keyword in Python), but I find that I can't do it consistently.  I appreciate consistency within the document.  Thanks.